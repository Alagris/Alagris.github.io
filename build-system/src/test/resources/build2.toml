
[[source]]
path = "sample.mealy"
[[source]]
path = "sample2.mealy"
