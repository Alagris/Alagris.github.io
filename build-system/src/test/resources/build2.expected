'011'
'10'
''
'a'
'b'
'z'
''
''
''
''
''
''
''
''
